frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -48,7 +48,7 @@ import { BubbleMark } from './bubble_mark';
 import { WriteInLine } from './write_in_line';
 import { HorizontalRule } from './horizontal_rule';
 import { ABSENTEE_TINT_COLOR } from '../config/globals';
-import { getBallotLayoutPageSize } from '../utils/get_ballot_layout_page_size';
+import { getBallotLayoutPageSizeStr } from '../utils/get_ballot_layout_page_size';
```

Comment:
Nit: FWIW, when reading and searching, I find abbreviations less useful/readable than the full strings: `getBallotLayoutPageSizeString`

Reply:
Also, "string" is not a useful description of what it is. Is it a name? A description? What is it for?